core/src/main/resources/benchmark-jgroups.xml

```diff
@@ -34,8 +34,6 @@
             <!-- dynamically add/remove protocols from stack -->
             <query value="remove-protocol=MERGE3"/>
             <query value="insert-protocol=MERGE2=above=PING"/>
-            <!-- insert custom protocol (just for illustration -->
-            <query value="insert-protocol=org.radargun.protocols.SLAVE_PARTITION=above=UDP"/>
```

Comment:
It broke the benchmark, I assumed it was because of the jGroups config schema change in the meantime, but on the second look I realize it is a custom protocol so it should be there. I will have a second look at this